axiom/datasets_integration_test.go

```diff
@@ -80,7 +80,7 @@ func (s *DatasetsTestSuite) SetupSuite() {
 	var err error
 	s.dataset, err = s.client.Datasets.Create(s.suiteCtx, axiom.DatasetCreateRequest{
 		Name:        "test-" + randString(),
```

Comment:
Could you prefix this with `axiom-go`? Makes it easier to figure where they belong.

Reply:
Sounds sane.